### PR TITLE
Add instructions to install font for hyperjs users

### PR DESCRIPTION
**Pull request by kartsims**
Added some instructions for hyperjs users as I spent 2 minutes looking for the solution :)

### CODE REVIEW

**Comment by avdv** [comment]
Hi @kartsims,

thanks for you PR and sorry for not responding for so long!

Are you sure that the instructions are correct? AFAICS, the name of the font should be "Hack Nerd Font", instead of "Hack Regular Nerd". If I use the latter, the system chooses an arbitrary font which supports the icons used by colorls, and those icons appear too small. It is the same behavior I see if I do not list any Nerd font in `fontFamily` at all.

**Comment by kartsims** [comment]
You are correct, my bad!

File: README.md [hunk 1]
```diff
@@ -122,6 +122,8 @@ Man pages have been added. Checkout `man colorls`.
 
     *Note for `iTerm2` users - Please enable the Nerd Font at iTerm2 > Preferences > Profiles > Text > Non-ASCII font > Hack Regular Nerd Font Complete.*
 
+    *Note for `HyperJS` users - Please add `"Hack Regular Nerd"` Font as an option to `fontFamily` in your `~/.hyper.js` file.*
```
Comment (avdv) [inline README.md:125]:
```suggestion
    *Note for `HyperJS` users - Please add `"Hack Nerd Font"` as an entry to `fontFamily` in your `~/.hyper.js` file.*
```
Is this OK?

Comment (kartsims) [reply README.md:125]: sounds good!

**Comment by avdv** [comment]
Thanks @kartsims !

**Pull request merged**